docs/installation_linux.rst

```diff
@@ -50,9 +50,11 @@ Install Poetry on Linux
 
 Poetry is used to manage Python dependencies of LISA.
 
-:warning: Please enter the root folder of LISA source code to run
-following commands to install poetry, since Poetry manages dependencies
-by the working folder.
+.. warning::
```

Comment:
Is there any tool can help on writting rst in vscode? If there is, document it in development.rst, and all of us can use it.

Reply:
I'm still learning and I'm currently using a VSCode extension [reStructuredText](https://marketplace.visualstudio.com/items?itemName=lextudio.restructuredtext). I will add that into doc.

Reply:
Added in VSCode section in setup.rst.

Reply:
BTW, we have files called installation and setup. The names sound similar, and confusing.

Reply:
I personally think they are distinguishing since installation refers to the source code and setup refers to environment etc. I changed the name of setup.rst to dev_setup.rst to make more contrast.